schemaTranslations.json

```diff
@@ -811,11 +811,11 @@
     "zh": "显示 PayPal"
   },
   "i18n.ShowVisa": {
-    "default": "Show Visa",
-    "fr": "Afficher Visa",
-    "it": "Mostra Visa",
-    "uk": "Показати Visa",
-    "zh": "显示 Visa"
+    "default": "Show Wire Transfer",
+    "fr": "Afficher virement bancaire",
+    "it": "Mostra Bonifico",
+    "uk": "Показати банківський переказ",
+    "zh": "显示 电汇"
```

Comment:
Do we have correct field name and translation values here?

Reply:
@BC-tymurbiedukhin Ahhh... think I messed up here, I will fix it and make a new pull request.